Apps/CocoaPods-FCM/src/View/DashboardView.swift

```diff
@@ -1,8 +1,102 @@
 import CioDataPipelines
+import CioMessagingInApp
 import SwiftUI
+import UIKit
```

Comment:
```suggestion
```

I think this import is only here because of the `UIViewRepresentable` being in this file. 

IMO, part of the acceptance criteria of this ticket is customers only need to interact with SwiftUI framework and not need to have to import `UIKit` in their code. So before merging, I think this import should be removed.

Reply:
Outdated. Removed, moved to SDK.